cmd/ib-sriov-cni/main.go

```diff
@@ -112,31 +113,26 @@ func cmdAdd(args *skel.CmdArgs) error {
 	if netConf.RdmaIso {
 		err = utils.EnsureRdmaSystemMode()
 		if err != nil {
-			return err
+			return nil, nil, err
 		}
 	}
 
 	netns, err := ns.GetNS(args.Netns)
```

Comment:
you don't use the netns you just return it. it will be better to just pass the args.Netns in Line 130